spec/index.html

```diff
@@ -351,6 +352,64 @@ <h3>Blank nodes</h3>
     </figure>
   </section>
 
+  <section id="section-triple-terms">
+    <h3>Triple terms</h3>
+
+    <p>As the basic unit of description in RDF, the triple is a simple
+    abstraction for stating a directed relationship between two resources.
+    Sometimes, we need to describe more detailed circumstances underlying such
+    relationships. While we could describe these using more sophisticated
+    modelling, there is a tension between simple, direct statements and more
+    complex, granular descriptions. This may require a trade-off between
+    directness and completeness.</p>
+
+    <p>To bridge this difference, we can use <a
+    data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a> as
+    <em>references</em> to simple statements. A triple term is a fixed object
+    that identifies a statement through its constituent subject, predicate and
+    object components.</p>
+
+    <p>This enables us to describe more concrete circumstances, such as events,
+    relationship entities or tokens in data sources, <em>in relation to</em>
+    the simple statements. That allows us to keep them as is, rather than
+    requiring a complete remodelling into more detailed structures, for the
+    sake of occasionally adding details.</p>
+
+    <p>Conceptually, we can <em>annotate</em> a simple statement with a
+    reference to something more detailed:</p>
+    <pre class="example nohighlight">
+      &lt;Bob&gt; &lt;is interested in> &lt;the Mona Lisa&gt; <sup><strong>~ &lt;Bob's interest></strong></sup> .
+    </pre>
+
+    <p>This annotation refers to Bob's interest as a concrete circumstance, with
+    a date of origin:</p>
+    <pre class="example nohighlight">
+      &lt;Bob's interest> &lt;is an> &lt;Interest> .
+      &lt;Bob's interest> &lt;since> &lt;4th of October 1998> .
+    </pre>
+
+    <p>Formally, an annotation is formed by two triples: the statement itself
+    and a <a data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying
+    triple</a>. This expresses the simple relationship and the reifying relation to that from the circumstance, as a concretization thereof:</p>
```

Comment:
This sentence took many readings (I think I still don't understand it) :-)
> This expresses the simple relationship and the reifying relation to that from the circumstance, as a concretization thereof

Reply:
Point taken; it got a bit twisted during a rewrite. How about:
> This expresses, on the one hand, the relationship as an abstract statement, and on the other, the circumstance as a concretization of that abstraction.

?